### PR TITLE
[PD] bump submodules `ot-br-posix`, `openthread` and `ot-nrf528xx ` to add PD Daemon service 

### PR DESCRIPTION
**Pull request by yangsong-cnyn**
ot-br-posix 6934ff024cc217dfd55aac8b21d6198d7013ed81
openthread 0fb1c22d2458a22c87be77f94aa5e2358313e83b 
ot-nrf528xx e3ea7ae20197d2f849aa237392cc58a9ca665a26

Add prefix delegation daemon service

### CODE REVIEW

File: script/otbr-setup.bash [hunk 1]
```diff
@@ -202,6 +202,9 @@ cmake --version
 
 pip3 install zeroconf
 
+apt-get install -y --no-install-recommends libgirepository1.0-dev
```
Comment (superwhd) [inline script/otbr-setup.bash:205]: What's depending on this?

Comment (yangsong-cnyn) [reply script/otbr-setup.bash:205]: The python PyGObject module relies on this C library

**Review by sunytt — APPROVED** [review]
LGTM

**Review by superwhd — APPROVED** [review]
LGTM 👍

**Review by jwhui — APPROVED** [review]

**Pull request merged**